### ISSUE TITLE
Check and fix all new features

### ISSUE THREAD

**Issue by karasu** [Antergos/Cnchi#359]
Check and fix (if necessary) all new added features

 - [X] LAMP / LEMP
 - [X] LTS Kernel

- I want to look into ways we can include them on the iso as well (let's leave this for 0.11.x)

**Comment by Mikaela** [comment]
Something is wrong with your checkboxes, they should be

* [ ] example
* [x] example2

(raw)

```
* [ ] example
* [x] example2
```

Also I am wondering if the new iso will also use [systemd-timesyncd](https://wiki.archlinux.org/index.php/Systemd-timesyncd) in addition to systemd-networkd.

**Comment by lots0logs** [comment]
I fixed them :smile: Though I always thought the syntax was the following:
```
  - [ ] example
  - [ ] example 2
```
That's how I've always done it. Oh well, it appears to work either way haha Cheers!

**Comment by karasu** [comment]
@Mikaela @lots0logs : Thanks both of you! :wink:

@Mikaela : I didn't know about it... I'm an old guy used to NTP, but I'm up to the challenge! (of changing it). Thanks again.

**Comment by karasu** [comment]
In 0.12 graphic drivers feature will be unavailable. Let's fix this for 0.14
